data.sql

```diff
@@ -14,4 +14,17 @@ VALUES
 	('Boarmon', '2005-06-7', 7, true, 20.4),
 	('Blossom', '1998-10-13', 3, true, 17),
 	('Ditto', '2022-05-14', 4, true, 22);
-	
\ No newline at end of file
+
+INSERT INTO owners(full_name, age)
+VALUES
+	('Sam Smith', 34),
+	('Jennifer Orwell', 19),
+	('Bob', 45),
+	('Melody Pond', 77),
+	('Dean Winchester', 14),
+	('Jodie Whittaker', 38);
+
+INSERT INTO species(name)
+VALUES
+	('Pokemon'),
+	('Digimon');
```

Comment:
- [ ] Again here we need to document the code that executes that if an animal's name ends with `mon` it should have `Digimon` as a `species_id`, all other animals should have `Pokemon` instead. Kindly do so

e.g
```
   UPDATE animals
   SET species_id = (SELECT id from species WHERE name = 'Digimon')
   WHERE name like '%mon';

   UPDATE animals
   SET species_id = (SELECT id from species WHERE name = 'Pokemon')
   WHERE species_id IS NULL;

```